vrml1.mac

```diff
@@ -1,33 +1,21 @@
 /control/verbose 2
 /run/verbose 2
-#/event/verbose 2
-#/tracking/verbose 2
-#
```

Comment:
missing run/initialize?
This didn't draw the event in the visualization after /run/beamOn 1 in the Qt interface. Right version worked for me.
I removed mostly things I don't understand. Again, you can change them back if you feel like it, I just try to follow KISS(keep it stupid, simple) less lines -> better